cups/ipp-support.c

```diff
@@ -740,9 +740,7 @@ ippAttributeString(
 
       case IPP_TAG_DATE :
           {
-            unsigned year;		/* Year */
-
-            year = ((unsigned)val->date[0] << 8) + (unsigned)val->date[1];
+            unsigned year = ((unsigned)val->date[0] << 8) | (unsigned)val->date[1];
```

Comment:
Still need a comment for the variable...

Reply:
year is pretty self explanatory though.